Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:0.10-onbuild
+
+VOLUME ["/project"]
+EXPOSE 6789
+
+CMD ["serve","/project/project.yml"]
```

Comment:
Does that mean that the project file *needs* to be named `project.yml`, or is it overridable?